### PR TITLE
Crc

### PR DESCRIPTION
**Pull request by husxk**
@kornelu123 can you look at this?

### CODE REVIEW

File: data_process.cpp [hunk 1]
```diff
@@ -0,0 +1,42 @@
+#include "data_process.hpp"
+
+#include <cstdio>
```
Comment (kornelu123) [inline data_process.cpp]: Isn't this include redundant ? I can't se us using it anywhere.

File: mt_rng.cpp [hunk 2]
```diff
@@ -54,7 +54,7 @@ uint32_t get_next(uint32_t min, uint32_t max)
     {
         return min;
     }
-    uint32_t range = (uint32_t)(max - min + 1);
+    uint64_t range = (uint32_t)(max - min + 1);
```
Comment (kornelu123) [inline mt_rng.cpp]: It's not right, it should be like : 
```
uint64_t range = (uint64_t) (max - min) + 1;
```

File: mt_rng.cpp [hunk 3]
```diff
@@ -81,4 +81,4 @@ uint32_t get_normal(double mean, double standard_deviation)
     } while (std::isnan(z0) || std::isnan(z1));
 
     return static_cast<uint32_t>(z0 * standard_deviation + mean);
-}
\ No newline at end of file
+}
```
Comment (kornelu123) [inline mt_rng.cpp:84]: I think you messed up some whitespaces, just FYI.

**Review by kornelu123 — COMMENTED** [review]
Seems fine. Few comments though.

File: data_process.cpp (hunk 1 above)
Comment (husxk) [reply data_process.cpp]: Correct, it was done for testing and I forgot to delete it.

File: mt_rng.cpp (hunk 2 above)
Comment (husxk) [reply mt_rng.cpp]: fixed

**Comment by kornelu123** [comment]
yeah, LGTM, lets merge it

**Comment by husxk** [comment]
Thanks for the review

**Pull request merged**